src/Reflection/ReflectionType.php

```diff
@@ -4,105 +4,50 @@
 
 namespace Roave\BetterReflection\Reflection;
 
-use Roave\BetterReflection\Reflection\Exception\ClassDoesNotExist;
-use Roave\BetterReflection\Reflection\Exception\ReflectionTypeDoesNotPointToAClassAlikeType;
-use Roave\BetterReflection\Reflector\Exception\IdentifierNotFound;
-use Roave\BetterReflection\Reflector\Reflector;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Name;
+use PhpParser\Node\NullableType;
+use PhpParser\Node\UnionType;
 
-use function array_key_exists;
-use function ltrim;
-use function strtolower;
-
-class ReflectionType
+abstract class ReflectionType
```

Comment:
We should skim the patch and look for any `|ReflectionType`, and drop it everywhere